backend/app/models/workflow_version_model.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import Computed, Integer, String, DateTime, JSON, ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from datetime import datetime, timezone
+from typing import List, Optional, Any
+from .base_model import BaseModel
+from .run_model import RunModel
+
+class WorkflowVersionModel(BaseModel):
```

Comment:
@preet-bhadra let's add a doc string here that clarifies that a version is created only when the workflow is run. The latest/current version is always in the WorkflowModel

Reply:
Added it.